src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: SMS OTP form best practices
+subhead: Learn how to optimize your SMS OTP form.
+authors:
+  - agektmr
+date: 2020-12-04
+hero: hero.jpg
+alt: A drawing of a woman using OTP to log in to a web app.
```

Comment:
I'm seeing a neon chat bubble sign photo, but it has a black margin that sticks out on larger screen. If you have another hero image as this alt suggests, let's try that one.

Reply:
Most of images I can find on the web are more square-ish than the wide one expected here hence I had to pad black to both sides. I created two new images. Which one do you think look better? I like the chat bubble one.
![hero2](https://user-images.githubusercontent.com/218144/100401123-f2698d80-309b-11eb-8b1f-512b47c11046.jpg)
![hero](https://user-images.githubusercontent.com/218144/100401128-f4cbe780-309b-11eb-867e-52d6fafe297a.jpg)